Dockerfile

```diff
@@ -1,21 +1,32 @@
-FROM quay.io/ukhomeofficedigital/centos-base
+FROM quay.io/ukhomeofficedigital/nodejs-base:v4.4.2
 
-RUN mkdir -p /opt/nodejs
-WORKDIR /opt/nodejs
+       # Copy downstream in which should help
+       # ensure everyone using this has a similar
+       # app structure.
+ONBUILD COPY . /app
```

Comment:
Not really.  If we want to do the rest of the onbuild work, the app really needs to be inside the container at that point.  The point is that the file structure should be the same for all our node apps, and if it's not the same, then you should build your image from the base image this uses.  For the record, I'm pretty sure this will work on all the hof apps.

Now with that said, I'd really rather the apps contents were not all dumped into the root of the git repo.  There's a whole bunch of files that would need to be added to the .dockerignore and I think our containers would be a lot cleaner if standarised on actually putting all the code we want to ship onto the container into a specific file.